tests/test_syntax/extensions/test_md_in_html.py

```diff
@@ -1517,6 +1517,24 @@ def test_md1_code_cdata(self):
             extensions=['md_in_html']
         )
 
+    def test_trialing_content_after_tag_in_md_block(self):
+
+        self.assertMarkdownRenders(
+            self.dedent(
+                """
+                <div markdown>
+                <div class="circle"></div>AAAAA<div class="circle"></div>
+                </div>
+                """
+            ),
+            '<div>\n'
+            '<div class="circle"></div>\n'
+            '<p>AAAAA<div class="circle"></p>\n'
```

Comment:
Specifying invalid HTML as expected result is a bit weird. Maybe at least add a comment like "Yes, it is invalid, but what we really test here is that trailing content is not lost"?

Reply:
Fair argument, I can do that.